.github/workflows/applications.yml

```diff
@@ -135,7 +135,7 @@ jobs:
     - name: Build
       run: cabal v2-build chainweb-node
     - name: Run Tests
-      run: cabal v2-run chainweb-tests -- --hide-successes --pattern='!/remote pact tests/ && !/Chainweb.Test.Pact.SPV/'
```

Comment:
@larskuhtz the remote pact tests were fixed with the latest PR bumping the retry rate to accommodate for the increase in polling time.